ocean/src/social.rs

```diff
@@ -17,7 +22,28 @@ impl SocialNetwork {
     * Result enum
     */
     pub fn add_friend(&mut self, crab_id: String, friend_id: String) -> Result<(), String> {
```

Comment:
What do you think about changing `crab_id` to a more generic name like `id` or `entity_id`? I think that could be nice to have in case some other beach has seals that also form a social network.